packages/engine/src/actions/message.js

```diff
@@ -0,0 +1,28 @@
+/*
+  Copyright 2020-2021 Lowdefy, Inc
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+function createMessage({ context }) {
+  return function message(params = {}) {
```

Comment:
@Gervwyk Should this be `message` or `displayMessage` like we have on context?

Reply:
I think displayMessage is more descriptive. Then it is the same since it will invoke the same method.

Reply:
I agree